client-sdk/ts-web/rt/src/accounts.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Unique module name.
+ */
+export const MODULE_NAME = 'accounts';
+
+export const CODE_INVALID_ARGUMENT = 1;
+export const CODE_INSUFFICIENT_BALANCE = 2;
+export const CODE_FORBIDDEN = 3;
+
+// Callable methods.
+export const METHOD_TRANSFER = 'accounts.Transfer';
+// Queries.
+export const METHOD_NONCE = 'accounts.Nonce';
+export const METHOD_BALANCES = 'accounts.Balances';
+
+export const CODE_TRANSFER_HEX = '00000001';
+export const CODE_BURN_HEX = '00000002';
+export const CODE_MINT_HEX = '00000003';
```

Comment:
What are these, events? Maybe call them `EVENT_*`? Also, why in hex, they could be integers?

Reply:
they're events. ok to the suggestions